### PR TITLE
Point community section to theforeman.org

### PR DESCRIPTION
**Pull request by ehelms**
This still has one issue to resolve: what to do about the scripts page. The RFC describes either:

 1) Add scripts section to Katello documentation itself
 2) Create a community tools repository

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
 Redirect /media/index.html https://theforeman.org/media.html
+
 Redirect /developers/bastion/ https://github.com/Katello/bastion#bastion-angularjs-based-foreman-ui-engine
 Redirect /developers/bastion/add_custom_module.html https://github.com/Katello/bastion#plugins
 Redirect /developers/bastion/add_entity.html https://github.com/Katello/bastion#basics-of-adding-a-new-entity
 Redirect /developers/bastion/dependencies.html https://github.com/Katello/bastion#dependencies
 Redirect /developers/bastion/i18n.html https://github.com/Katello/bastion#i18n
+
+Redirect /community/index.html https://theforeman.org/support.html
+Redirect /community/index.html#contribute https://theforeman.org/contribute.html
```
Comment (johnpmitsch) [inline .htaccess]: Do you need this line?

Comment (ehelms) [inline .htaccess]: Yes - its not a comment, its linking a section specifically to another page.

Comment (johnpmitsch) [inline .htaccess]: that makes sense now :)

**Comment by johnpmitsch** [comment]
ACK

**Pull request merged**